env0/data_github_installation_id.go

```diff
@@ -0,0 +1,46 @@
+package env0
+
+import (
+	"context"
+
+	"github.com/env0/terraform-provider-env0/client"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataGithubInstallationId() *schema.Resource {
+	return &schema.Resource{
+		Description: "returns the github installation id of a git hub repositroy",
+
+		ReadContext: dataGithubInstallationIdRead,
+
+		Schema: map[string]*schema.Schema{
+			"repository": {
+				Type:        schema.TypeString,
+				Description: "the name of the repository",
+				Required:    true,
+			},
+			"github_installation_id": {
+				Type:        schema.TypeInt,
+				Description: "the github installation id",
+				Computed:    true,
+			},
```

Comment:
why both? "repository" is parameter and "github installation id" is the return type?

Reply:
correct. One is an input, the other is computed.